src/classes/MyHandler/MyHandler.ts

```diff
@@ -1173,7 +1173,11 @@ export default class MyHandler extends Handler {
         const isInPricelist =
             ourItemsHVCount > 0 // Only check if this not empty
                 ? Object.keys(getHighValue.our.items).some(sku => {
-                      return checkExist.getPrice(sku, false) !== null; // Return true if exist in pricelist, enabled or not.
+                      // look at the high value items and for the given sku see if any have a price enabled for the 'high value sku'
+                      const assetidPriceEnabled = items['our'][sku].some(
+                          item => checkExist.getPrice(item.id, false) !== null
+                      );
+                      return checkExist.getPrice(sku, false) !== null || assetidPriceEnabled; // Return true if exist in pricelist, enabled or not.
                   })
                 : null;
 
```

Comment:
So I think the best way is to add something above L1173, for example

```js
let isAssetidPriceEnabled = false
```
Then at L1179, we can add
```js
if (assetidPriceEnabled) isAssetidPriceEnabled = true
````

Reply:
this was culled